src/eth1spec/spec.py

```diff
@@ -35,6 +35,8 @@
 from .evm.interpreter import process_call
 
 BLOCK_REWARD = 5 * 10 ** 18
```

Comment:
*nitpick*

Not part of this PR but, some alternate ideas that come to mind for this variable.

```python
BLOCK_REWARD = 5000000.....00000  # 5 * 10**18
# or
BLOCK_REWARD = 5 * (10 ** 18)
# or
ONE_ETHER = 1**18
BLOCK_REWARD = 5 * ONE_ETHER
```

Reply:
I like `BLOCK_REWARD = 5 * ONE_ETHER`, it's more readable

Reply:
I do as well.